### PR TITLE
Add some more tabletop fluff

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Content "Added more tabletop games, by u/MuscleAgile5442 aka Cosmic"

#### Purpose of change

add some more board games into the game

#### Describe the solution

<details>
  <summary>added some tabletop games:</summary>

- Senet
- Mahjong
- Chaturanga Chess
- Backgammon
- Latrunculi
- Go
- Shogi
- Hnefatafl
- Alquerque
- Domino
- Jack Straws
- Snakes And Ladders
- Ludo
- Pop-Up Pirate
- Ambuscade! (aka Ambush)
- Patrol Leader (aka Ranger)
- War Tanks (aka Open Fire)
</details>

added them to existed loot groups

added oriental games to dojo

added Hnefatafl to Norse Evangelical trader (because that sounds fun)

#### Testing

Game works, loot spawns, i guess the density test will fail, because most of them are boxes with a lot of empty space

#### Additional context

All items were maded by Cosmic#8484 aka u/MuscleAgile5442, i just polished it for a bit

### CODE REVIEW

File: data/json/items/fluff.json [hunk 1]
```diff
@@ -239,6 +239,278 @@
...
+    "color": "green",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "ranger",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str_sp": "Patrol Leader" },
+    "description": "Patrol Leader is a tactical solitaire board game.  Is this game, the player is put in the role of a patrol leader in different modern battlefield scenarios, performing missions, capturing or killing enemies, protecting its teammates and advancing in the map. Very accurate and realistic, guaranteed fun if you like board games.",
```
Comment (MNG-cataclysm) [inline data/json/items/fluff.json:487]:
```suggestion
    "description": "Patrol Leader is a tactical solitaire board game.  Is this game, the player is put in the role of a patrol leader in different modern battlefield scenarios, performing missions, capturing or killing enemies, protecting its teammates and advancing in the map.  Very accurate and realistic, guaranteed fun if you like board games.",
```
Final spacing error to fix.

**Review by MNG-cataclysm — CHANGES_REQUESTED** [review]
There are several errors caused by several items being denser than they should. I'd check the material entry which the board games use, and use the d = m/v equation to recalculate, either lowering the mass or increasing the volume.

(d = m/v is density = mass/volume)

File: data/json/items/fluff.json [hunk 2]
```diff
@@ -239,6 +239,278 @@
...
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "mahjong",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "mahjong set" },
+    "description": "A very beautiful, and really huge plastic box, that contain all the pieces needed to play a game of mahjong, and also a extremely small, apparently poorly translated but comprehensible guide of how to play it.  Mahjong is a tile-based game where you need to find pairs of pieces and take the biggest score.",
+    "weight": "5530 g",
```
Comment (MNG-cataclysm) [inline data/json/items/fluff.json:264]:
```suggestion
    "weight": "4000 g",
```
This was the last thing that gave a density error. Lowering the weight will fix this.

**Review by MNG-cataclysm — CHANGES_REQUESTED** [review]

File: data/json/items/fluff.json [hunk 3]
```diff
@@ -239,6 +239,278 @@
...
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "shogi",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "shogi set" },
+    "description": "A wooden box containing all the pieces needed to play a game of shogi, and also a extremely small guide of how to play it, with some letters in Japanese.  Also known as Japanese chess, it is a strategy board game for two players.  It was once one of the most popular board games in Japan, before the Cataclysm, and the main objective is to, though strategy, tsumi (checkmate) the opponent's osho (king).",
+    "weight": "2871 g",
+    "volume": "3033 L",
```
Comment (GuardianDll) [inline data/json/items/fluff.json:345]:
```suggestion
    "volume": "3033 ml",
```

File: data/json/items/fluff.json [hunk 4]
```diff
@@ -239,6 +239,278 @@
...
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "shogi",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "shogi set" },
+    "description": "A wooden box containing all the pieces needed to play a game of shogi, and also a extremely small guide of how to play it, with some letters in Japanese.  Also known as Japanese chess, it is a strategy board game for two players.  It was once one of the most popular board games in Japan, before the Cataclysm, and the main objective is to, though strategy, tsumi (checkmate) the opponent's osho (king).",
+    "weight": "2871 g",
```
Comment (GuardianDll) [inline data/json/items/fluff.json:344]:
```suggestion
    "weight": "2571 g",
```

**Comment by MNG-cataclysm** [comment]
If you could, I'd recommend kicking the Ubuntu test. 
@Maleclypse If GuardianDll can't, could you restart the GCC 11 test, please?

**Review by MNG-cataclysm — APPROVED** [review]

**Pull request merged**